File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/GsrBuildingHolder.java
package com.pennapps.labs.pennmobile;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Created by Andrew on 11/5/2017.
 * RecyclerView ViewHolder for a specific building with GSRs.
 */

public class GsrBuildingHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.gsr_building_name) TextView gsrBuildingName;
    @BindView(R.id.gsr_availability_in_building) RecyclerView recyclerView;

    public GsrBuildingHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/GsrBuildingHolder.kt
package com.pennapps.labs.pennmobile

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import kotlinx.android.synthetic.main.gsr_building.view.gsr_availability_in_building
import kotlinx.android.synthetic.main.gsr_building.view.gsr_building_name

class GsrBuildingHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    internal var gsrBuildingName: TextView? = null
    internal var recyclerView: RecyclerView? = null

    init {
        recyclerView = itemView.gsr_availability_in_building
        gsrBuildingName = itemView.gsr_building_name
    }
}
